deepinv/training_utils.py

```diff
@@ -529,6 +736,22 @@ def test(
     return test_psnr, test_std_psnr, linear_psnr, linear_std_psnr
 
 
+def train(*args, **kwargs):
+    """
+    WARNING: DEPRECATED.
```

Comment:
we can use

.. warning: 
   
    This usage is deprecated and might be removed in future versions of deepinv.